src/pages/users.vue

```
@@ -0,0 +1,156 @@
+    isEditing.value = true;
+    dialog.value = true;
+};
+
+const deleteItem = async (item: User) => {
+    try {
+        if (item.uuid) {
+            await deleteUser(item.uuid);
+            users.value = users.value.filter(user => user.uuid !== item.uuid);
+        }
+        snackbar.Success("Korisnik je uspješno obrisan")
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)